.github/workflows/all_solutions.yml

```diff
@@ -443,6 +443,21 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Setup .NET Core 2.2.103
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '2.2.103'
+
+      - name: Setup .NET Core 3.1.100
```

Comment:
3.1.100 is already setup:
https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#net-core-sdk